src/expr/src/function/args.rs

```diff
@@ -15,24 +15,22 @@
 use std::slice;
 
 use risingwave_common::types::DataType;
-use risingwave_common::util::sort_util::ColumnOrder;
-use risingwave_expr::expr::{AggKind, ExpressionRef};
 
 /// An aggregation function may accept 0, 1 or 2 arguments.
 #[derive(Clone, Debug)]
-pub enum AggArgs {
-    /// `None` is used for aggregation function accepts 0 arguments, such as `count(*)`.
+pub enum FuncArgs {
```

Comment:
This struct seems not general enough to be named `FuncArgs`. 👀 For example, there are functions that take more than 2 arguments, and not all argument is a simple `InputRef` that can be denoted as an index.

If you mean that this will also be used by `OverWindow`, I guess it's still reasonable to keep the name `AggArgs`. `OverWindow` is known as "Over Aggregation" as well.